challenge.md

```diff
@@ -22,6 +29,12 @@
 </main>
 ```
 
+`const header = document.querySelector("h2.subtitle");`
+`const text = document.createTextNode("The MERN stack");`
+`header.textContent = '';`
+`header.appendChild(text);`
+`header.classList.remove("subtitle");`
```

Comment:
Great implementation! 👏🏼

There is a more simple solution in this case. Instead of using `createTextNode` method, you can directly act on `textContent` property to change text on the fly.

```javascript
const header = document.querySelector(".subtitle");
header.textContent = "The MERN stack";
```